flash_examples/predict/style_transfer.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import flash
+from flash.data.utils import download_data
+from flash.utils.imports import _PYSTICHE_AVAILABLE
+from flash.vision.style_transfer import StyleTransfer
+
+if _PYSTICHE_AVAILABLE:
+    import pystiche.demo
+else:
+    print("Please, run `pip install pystiche`")
+    sys.exit(0)
+
+download_data("http://images.cocodataset.org/zips/train2014.zip", "data")
+
+data_module = ImageUnsupervisedData.from_folder("data")
```

Comment:
I need something like a `ImageUnsupervised` class. If that is not possible, we could also use any other data loading class that returns all images. In that case, I need to know how to discard the class labels or annotations.

Reply:
what are usually the inputs/labels for style transfer - pair of images ? We are about to introduce a similar data loading api for semantic segmentation that might be extend to other image generation tasks. /cc @ethanwharris 

@pmeier https://github.com/PyTorchLightning/lightning-flash/pull/239/files#diff-fff239f44797f0092593a14bf1af9a4b97d1959cba4a6a759eadbf126a697fc5R41

Reply:
> what are usually the inputs/labels for style transfer - pair of images ?

Style transfer is unsupervised, so there are not pairs. I just need the images.